### PR TITLE
DTSSTCI-865

### PR DESCRIPTION
**Pull request by tomxelliott**
### Change description ###
Add death of appellant as close case reason

### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSSTCI-865

Screenshot 2024-06-13 at 09 45 22

Screenshot 2024-06-13 at 09 45 58

### CODE REVIEW

**Review by oleon23 — DISMISSED** [review]

**Review by pr00279 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/State.java [hunk 1]
```diff
@@ -98,6 +98,13 @@ public enum State {
     )
     ConsentOrder("ConsentOrder"),
 
+    @CCD(
+        label = "Death of Appellant",
+        hint = "### ${cicCaseFullName}\nCase number: ${hyphenatedCaseRef}",
+        access = {DefaultStateAccessExcludingCAA.class}
+    )
+    DeathOfAppellant("DeathOfAppellant"),
```
Comment (rajaram-nagarajan) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/State.java:106]: I think only the case close reason has been changed. The state should still be CaseClosed.

Comment (tomxelliott) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/State.java:106]: It is the design approach the old team used for the reason. The state will not be used

**Review by rajaram-nagarajan — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseWithdrawnNotificationTest.java [hunk 2]
```diff
@@ -26,6 +25,9 @@
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.sptribs.caseworker.model.CloseReason.DeathOfAppellant;
+import static uk.gov.hmcts.sptribs.caseworker.model.CloseReason.Withdrawn;
+import static uk.gov.hmcts.sptribs.common.CommonConstants.DEATH_OF_APPELLANT_EMAIL_CONTENT;
```
Comment (pr00279) [inline src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseWithdrawnNotificationTest.java:30]: Only a small formatting thing - is this import really needed when CommonConstants is already imported? The others are called via CommonConstants.CONSTANT_NAME - for consistency should this not be the same?

Comment (tomxelliott) [reply src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseWithdrawnNotificationTest.java:30]: Sure can use static imports for all common constants

**Review by rajaram-nagarajan — APPROVED** [review]

**Pull request merged**